### PR TITLE
Update legal name and change FAQ -> Help

### PR DESCRIPTION
**Pull request by salbertson**

### CODE REVIEW

File: app/views/application/_footer.haml [hunk 1]
```diff
@@ -15,16 +15,15 @@
...
 
   %p
-    Hound is maintained by #{link_to "thoughtbot", "https://thoughtbot.com/"}.
-    Questions? Try the
-    #{link_to "FAQ", ENV.fetch("FAQ_URL")}
+    Questions? Try
+    #{link_to "Help", ENV.fetch("HELP_URL"), new_window_options}
     or tweet
-    #{link_to "@houndci", "https://twitter.com/houndci", new_window_options}.
+    #{link_to "@houndci", "https://twitter.com/share?text=@houndci &url=\"\"", new_window_options}.
```
[the remark this answers is not in the document]
Comment (gylaz) [reply app/views/application/_footer.haml]: Is that `url` param necessary?

File: config/routes.rb [hunk 2]
```diff
@@ -15,6 +15,7 @@
   get "/sign_out", to: "sessions#destroy"
   get "/configuration", to: "pages#configuration"
   get "/faq", to: redirect(ENV.fetch("FAQ_URL"), status: 302)
```
Comment (gylaz) [inline config/routes.rb]: Should this redirect to help?

File: app/views/application/_footer.haml (hunk 1 above)
[the remark this answers is not in the document]
Comment (salbertson) [reply app/views/application/_footer.haml]: Yea, we get this if an empty `url` isn't included.

screenshot 2018-05-30 10 38 41

File: config/routes.rb (hunk 2 above)
Comment (salbertson) [reply config/routes.rb]: I don't think so, we still have a FAQ section.

File: app/views/pages/configuration.haml [hunk 3]
```diff
@@ -625,6 +625,6 @@
 
       %p
         Please tweet at us
-        = link_to "@houndci", "https://twitter.com/houndci", new_window_options
-        or email us at #{mail_to "hound@thoughtbot.com"}
+        = link_to "@houndci", "https://twitter.com/share?text=@houndci &url=\"\"", new_window_options
+        or email us at #{link_to "hello@houndci.com", "javascript:window.Intercom('showNewMessage', 'I have a quick question.');"}
```
[the remark this answers is not in the document]
Comment (zamberi09) [reply app/views/pages/configuration.haml:629]: https://zamberi09.com

**Pull request merged**